devel/doctest/Portfile

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem 1.0
+PortGroup cmake 1.1
+PortGroup github 1.0
+github.setup        onqtam doctest 2.4.6
+version             2.4.6
```

Comment:
remove the `version` field, it's redundant as it is already set from the line above by the github PG